src/analytics.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import fs from 'fs-extra'
+import { default as OS } from 'os'
+import type Config from 'cli-engine-config'
+import HTTP from 'cli-engine-command/lib/http'
+import Netrc from 'netrc-parser'
+import Output from 'cli-engine-command/lib/output'
+import URI from 'url'
+
+export default class AnalyticsCommand {
+  command: string
+  plugin: string
+  pluginVersion: string
+  version: string
+  platform: string
+  arch: string
+  start: any
+  config: Config
+
+  constructor (command: string, plugin: string, pluginVersion: string, config: Config) {
+    this.config = config
+    this.command = command
+    this.plugin = plugin
+    this.pluginVersion = pluginVersion
+    this.platform = OS.platform()
+    this.arch = OS.arch()
+  }
+
+  get version (): string {
+    return this.config.version
+  }
+
+  async recordEnd () {
+    if (AnalyticsCommand.skipAnalytics() || process.argv.length < 2) {
```

Comment:
that was literally copied over verbatim from the Go code. I'm not even sure the logic around it.